siri/acsb/acsb_accessibility-v0.3.xsd

```diff
@@ -29,7 +29,7 @@
 				<Publisher>Kizoom, 109-123 Clifton Street, London EC4A 4LD</Publisher>
 				<Rights>Unclassified
        <Copyright>CEN, Crown Copyright 2006-2012</Copyright>
-    </Rights>
+				</Rights>
```

Comment:
This unrelated change of a siri file should be removed

Reply:
removed